components/klaudsolcms/modals/modal_body/CollectionTypeBody.js

```diff
@@ -1,65 +1,65 @@
-import { useState, useContext } from 'react';
-import {Formik, Form, Field} from 'formik';
-import { loadEntityTypes } from '@/components/reducers/actions';
-import RootContext from '@/components/contexts/RootContext';
+import { useState, useContext } from "react";
+import { Formik, Form, Field } from "formik";
+import { loadEntityTypes } from "@/components/reducers/actions";
+import RootContext from "@/components/contexts/RootContext";
 
-export default function CollectionTypeBody({formRef}) {
-    const { state: rootState, dispatch: rootDispatch } = useContext(RootContext);
+export default function CollectionTypeBody({ formRef }) {
+  const { state: rootState, dispatch: rootDispatch } = useContext(RootContext);
 
-    const formikParams = {
-      initialValues: {},
-      innerRef: formRef,
-      onSubmit: (values) => {
-        console.error(JSON.stringify(values));
+  const formikParams = {
+    initialValues: {},
+    innerRef: formRef,
+    onSubmit: (values) => {
+      console.error(JSON.stringify(values));
 
-        (async () => {
-          try {
+      (async () => {
+        try {
           //refactor to reducers/actions
-           await fetch(`/api/entity_types`, {
-              method: 'POST',
-              body: JSON.stringify(values),
-              headers: {
-                'Content-Type': 'application/json'
-              }
-            });
-          } catch (error) {
+          await fetch(`/api/entity_types`, {
+            method: "POST",
+            body: JSON.stringify(values),
+            headers: {
+              "Content-Type": "application/json",
+            },
+          });
+        } catch (error) {
+        } finally {
+          await loadEntityTypes({ rootState, rootDispatch });
+        }
+      })();
+    },
+  };
 
-          } finally {
-            await loadEntityTypes({rootState, rootDispatch});
-          }
-        })();
-      }
-    };
-    return ( 
+  return (
     <>
-        <Formik {...formikParams}>
-          <Form>
+      <Formik {...formikParams}>
+        <Form>
           <div>
-              <div className="d-flex justify-content-between align-items-center">
+            <div className="d-flex justify-content-between align-items-center">
               <h6 className="mx-3"> Configurations </h6>
               <div>
-              <button className="btn_modal_settings"> Basic settings </button>
-  
+                <button className="btn_modal_settings"> Basic settings </button>
               </div>
-            
-              </div>
-              <div className="block_bar"></div>
-              <div className="row">
-                <div className="col">
+            </div>
+            <div className="block_bar"></div>
```

Comment:
If there are no contents inside a `div`, you can close it directly.

```suggestion
            <div className="block_bar" />
```